testsuite/features/init_clients/srv_create_activationkey.feature

```diff
@@ -9,13 +9,28 @@ Feature: Create activation keys
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  Scenario: Clone the child custom channel including test repositories
```

Comment:
Could you explain why we need to clone this channel?

Reply:
The initial idea was to avoid creating a new channel and syncing it again, and clone directly the initial test channel with everything that we need in it, it saves some steps and some time.
However the result would be the about the same by not cloning (apart the names of patches) so we can do that instead.

Reply:
Didn't see the second part, sorry
It seems synchronized base products cannot be assigned as child-channels so the test channel needs to be a child

Reply:
Not cloning would imply adding scenarios from https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/core/srv_create_repository.feature#L13 to https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/core/srv_create_repository.feature#L51 in order to recreate the channel
To be discussed to see if it is worth it

Reply:
From my point of view yup, it's worth to do it that way.

Reply:
That also means repeating it for every activation key we want to associate to a base product in the future
I'll apply it for now, but it might need to be optimized in the future